TarkovBot.py

```diff
@@ -0,0 +1,68 @@
+import discord
+from config_bot import TOKEN
+import pandas as pd
+from discord.ext import commands
+pd.options.mode.chained_assignment = None
+
+intents = discord.Intents().all()
```

Comment:
@JachymDolezal Maybe consider encapsulating all the attributes to a class and have the following methods there. You can use asab.service and application like: https://github.com/TeskaLabs/asab/blob/master/examples/task.py#L5